aptos-move/writeset-transaction-generator/src/writeset_builder.rs

```diff
@@ -118,7 +118,7 @@ where
         TimedFeatures::enable_all(),
     )
     .unwrap();
-    let state_view_storage = StorageAdapter::new(state_view);
+    let state_view_storage: StorageAdapter<_, ()> = StorageAdapter::new(state_view);
```

Comment:
This file seems unused to me and we keep modifying it on new features every time, I am wondering if it is dead code or we actually use it somewhere? Maybe @runtian-zhou @zekun000 know?